src/PlusDataCollection/PlusSerialLine.cxx

```diff
@@ -235,3 +235,47 @@ unsigned int SerialLine::GetNumberOfBytesAvailableForReading() const
   return 0;
 #endif
 }
+
+PlusStatus SerialLine::SetDTR(bool onOff)
+{
+#ifdef _WIN32
+  DWORD dwFunc = CLRDTR;
+  if (onOff)
+  {
+    dwFunc = SETDTR;
+  }
+  if (EscapeCommFunction(CommHandle, dwFunc))
+  {
+    return PLUS_SUCCESS;
+  }
+  else
+  {
+    return PLUS_FAIL;
+  }
+#else
+  LOG_ERROR("SerialLine::SetDTR() is only implemented on Windows");

```

Comment:
Ah yes I see. I may try to refactor to use the ndicapi interface, as it does all of the serial business for us.